src/pages/basic_data/app_admin/central_import/DuplicateSearchFilterForm.tsx

```diff
@@ -0,0 +1,270 @@
+                  {({ field }: FieldProps) => (
+                    <TextField
+                      data-testid="duplicate-search-doi-textfield"
+                      variant="outlined"
+                      {...field}
+                      fullWidth
+                      multiline
+                      disabled={!formikProps.values.isDoiChecked}
+                    />
+                  )}
+                </Field>
+              </StyledFormElementWrapper>
+
+              <StyledCenterWrapper>
+                <Typography>{t('central_import.or_search_with')}</Typography>
+              </StyledCenterWrapper>
+
+              <StyledFormElementWrapper>
+                <Field
+                  as={StyledFormControlLabel}
+                  type="checkbox"
+                  name="isTitleChecked"
+                  control={<Checkbox />}
+                  label={t('central_import.title')}
+                  onClick={(event: any) => {
+                    if (event.target.value) {
+                      formikProps.setFieldValue('isDoiChecked', false);
+                    }
+                  }}
+                />
+                <Field name={'title'}>
+                  {({ field }: FieldProps) => (
+                    <TextField
+                      data-testid="duplicate-search-title-textfield"
+                      fullWidth
+                      {...field}
+                      variant="outlined"
+                      disabled={!formikProps.values.isTitleChecked}
+                      multiline
+                    />
+                  )}
+                </Field>
+              </StyledFormElementWrapper>
+
+              <StyledFormElementWrapper>
+                <Field
+                  as={StyledFormControlLabel}
+                  type="checkbox"
+                  name="isAuthorChecked"
+                  control={<Checkbox />}
+                  label={t('central_import.author')}
+                  onClick={(event: any) => {
+                    if (event.target.value) {
+                      formikProps.setFieldValue('isDoiChecked', false);
+                    }
+                  }}
+                />
+                <Field name={'author'}>
+                  {({ field }: FieldProps) => (
+                    <TextField
+                      data-testid="duplicate-search-author-textfield"
+                      fullWidth
+                      {...field}
+                      variant="outlined"
+                      disabled={!formikProps.values.isAuthorChecked}
+                      multiline
+                    />
+                  )}
+                </Field>
+              </StyledFormElementWrapper>
+
+              <StyledFormElementWrapper>
+                <Field
+                  as={StyledFormControlLabel}
+                  type="checkbox"
+                  name="isIssnChecked"
+                  control={<Checkbox />}
+                  label="ISSN"
```

Comment:
i18n? (selv om det blir det samme for begge)

Reply:
absolutt. bare avglemt. fikset